src/js/tracks/html-track-element-list.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file html-track-element.js
+ */
+
+import * as browser from '../utils/browser.js';
+
+class HtmlTrackElementList {
+  constructor(trackElements = []) {
+    this.trackElements_ = [];
+
+    Object.defineProperty(this, 'length', {
```

Comment:
unfortunately, for IE8, we need to use a custom object because IE8 only lets use `defineProperty` on dom elements.
See https://github.com/videojs/video.js/blob/master/src/js/tracks/text-track.js#L43-L50 for a template.